src/Modules/Content/Dnn.PersonaBar.Pages/Pages.Web/src/components/SearchPage/SearchResult.jsx

```diff
@@ -4,75 +4,129 @@ import GridCell from "dnn-grid-cell";
+                                        updateSearchAdvancedTags={this.props.updateSearchAdvancedTags}
+                                        filterByWorkflow={this.props.filterByWorkflow}  />
+                                );})}
+                        </LazyLoad>
+                            
+                            <div>
+                            {/* {searchList.map((item,index) => {
+                                return (
+                                    <SearchResultCard key={index}
+                                        item={item}
+                                        clearSearch={this.props.clearSearch}
+                                        onLoadPage={this.props.onLoadPage}
+                                        buildBreadCrumbPath={this.props.buildBreadCrumbPath}
+                                        setEmptyStateMessage={this.props.setEmptyStateMessage}
+                                        tags={this.props.tags}
+                                        onSearch={this.props.onSearch}
+                                        onViewPage={this.props.onViewPage}
+                                        onViewEditPage={this.props.onViewEditPage}
+                                        CallCustomAction={this.props.CallCustomAction}
+                                        getPageTypeLabel={this.props.getPageTypeLabel}
+                                        getPublishStatusLabel={this.props.getPublishStatusLabel}
+                                        filterByPageType={this.props.filterByPageType}
+                                        filterByPublishStatus={this.props.filterByPublishStatus}
+                                        updateFilterByPageStatusOptions={this.props.updateFilterByPageStatusOptions}
+                                        updateFilterByPageTypeOptions={this.props.updateFilterByPageTypeOptions}
+                                        updateFilterByWorkflowOptions={this.props.updateFilterByWorkflowOptions}
+                                        updateFilterStartEndDate={this.props.updateFilterStartEndDate}
+                                        startDate={this.props.startDate}
+                                        endDate={this.props.endDate} 
+                                        pageInContextComponents={this.props.pageInContextComponents} 
+                                        pageTypeSelectorComponents={this.props.pageTypeSelectorComponents}
+                                        updateSearchAdvancedTags={this.props.updateSearchAdvancedTags}
+                                        filterByWorkflow={this.props.filterByWorkflow}  />
+                                );})} */}
+                            </div>
+                        </div>
                     </GridCell>
                 </GridCell>
             </GridCell>
-        );
+        :<div></div>);
```

Comment:
I almost thought the colon was a typo. I think using ternary operator with lengthy JSX is not very readable.